src/components/portal/portal.pw.tsx

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { test, expect } from "@playwright/experimental-ct-react17";
+import TestComponent from "./components.test-pw";
+import { sidebarPreview } from "../../../playwright/components/sidebar";
+
+test.describe("tests for inertOptOut prop on Portal component", () => {
+  test("with the inertOptOut prop true, a button in the portal can be clicked when a modal is opened", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<TestComponent inertOptOut />);
+
+    await page.getByRole("button", { name: "Open sidebar" }).click();
+    await expect(sidebarPreview(page)).toBeVisible();
+
+    // no assertion here - but this click action will cause a test failure if the button can't be found or is
+    // covered by another element (such as the modal overlay)
+    await page.getByRole("button", { name: "Test for inertOptOut" }).click();
```

Comment:
Unfortunately this won't test for the right thing - I'm already testing in the unit tests for `useModalAria` that the element is in the accessibility tree, which would fail if it was `aria-hidden="true"`.

The issue I was trying to guard against here was one I came upon in an earlier stage of development, where the element exists and is not `inert` or `aria-hidden`, but can't be phsyically clicked due to being behind the modal overlay. I don't think `toBeHidden` covers that unfortunately (although I can try that out).

Reply:
fair, I agree it would be nice to assert something 😄 But surely you mean check that it *has* been focused?

Reply:
Ah yes sorry! 😄